yarn.lock

```diff
@@ -1,10 +1,24 @@
 # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
you could do a rebase with master instead of a merge, and if you do a ``git rebase -i master`` you can group and squash similar commits as well, such as the "snapshot update" ones

Reply:
I wasn't sure what I was doing when I had to include the most recents commits from master (containing the tooltip fixes) to my working branch, so I was afraid of messing up the commits history. Should I do a rebase now?